src/packages/README.md

```diff
@@ -23,8 +23,8 @@ to check what packages are available for a specific ABI version.
 
 | ABI        | #       | Manager | Install path   | Suitable for     | Examples |
 |------------|---------|---------|----------------|------------------|----------|
-| Hybrid ABI | ~20,000 | `pkg64` | `/usr/local64` | Day-to-day use   | bash<br/>gdb-cheri<br/>llvm-base<br/>vim |
-| CheriABI   | ~8,000  | `pkg64c`| `/usr/local`   | Experimental use | git<br/>nano<br/>rsync<br/>sudo |
+| Hybrid ABI | ~24,000 | `pkg64` | `/usr/local64` | Day-to-day use   | bash<br/>gdb-cheri<br/>llvm-base<br/>vim |
```

Comment:
If we have a working CheriABI bash, should we remove bash from the examples here?

Reply:
That's a good idea. I've had to remove examples from the table because it didn't look good after I added the benchmark ABI. I've updated examples in ABI-specific sections.